### PR TITLE
Gsoc2024 post

### PR DESCRIPTION
**Pull request by eimrek**
hi @unkcpz, @GeigerJ2 

Here's the post about GSOC 2024. Would you add anything extra?

Also, in principle, the project ended at the end of August. We could possibly backdate this to mid September, but i guess it's better to just put today's date.

### CODE REVIEW

File: docs/news/posts/2024-10-17-google-summer-of-code.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+blogpost: true
+category: News
+tags: summer-of-code
+date: 2024-10-17
+---
+
+# AiiDA at Google Summer of Code 2024
+
+Although the Google Summer of Code project has ended for this year, we are proud to announce that AiiDA has successfully participated (under the [NumFOCUS](https://numfocus.org/) organization umbrella). Our summer student [Sharan Poojari](https://github.com/SharanRP) built a prototype for a [next-generation web-based AiiDA provenance explorer](https://github.com/aiidateam/aiida-explorer).
```
Comment (unkcpz) [inline docs/news/posts/2024-10-17-google-summer-of-code.md:10]:
```suggestion
We are proud to announce that AiiDA has successfully participated (under the [NumFOCUS](https://numfocus.org/) organization umbrella). 
Our summer student [Sharan Poojari](https://github.com/SharanRP) built a prototype for a [next-generation web-based AiiDA provenance explorer](https://github.com/aiidateam/aiida-explorer).
```

File: docs/news/posts/2024-10-17-google-summer-of-code.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+<img src="../pics/2024-gsoc-aiida-explorer.png" alt="2024-gsoc-aiida-explorer" height="500px">
+
+Thank you, Sharan!
+
+For more details, see the [project report](https://github.com/aiidateam/aiida-explorer/blob/gsoc/gsoc/README.md). For a list of initially proposed projects, see [here](https://github.com/aiidateam/aiida-core/wiki/GSoC-2024-Projects).
```
Comment (unkcpz) [inline docs/news/posts/2024-10-17-google-summer-of-code.md:16]:
```suggestion
For more details of the outcome, see the [project report](https://github.com/aiidateam/aiida-explorer/blob/gsoc/gsoc/README.md).
```

**Review by unkcpz — APPROVED** [review]
Thanks @eimrek!
I think it is fine to use the date today. 

Just two minor request to make the post more compact.

**Comment by eimrek** [comment]
thanks @unkcpz

**Pull request merged**